python/veles/objects/chunk.py

```diff
@@ -15,6 +19,15 @@ def __init__(self, client, proto_obj=None, id_path=None, parent=None):
         self._end = proto_obj.chunk_end
         self._blob = parent.get_blob() if parent is not None else None
         self._cursor = self._start
+        self.items = [ChunkDataItem(item) for item in self._proto_obj.items]
+
+    def _from_another(self, other):
+        super(Chunk, self)._from_another(other)
+        self.items = other.items
+
+    def items_from_another(self, other):
```

Comment:
Yes it should be private - I'll change it.
About name and copying - I created this method to "transfer" chunk items from object received from network to normal chunk, and in this use-case the network object isn't used afterwards. I can rename it to something like "_transfer_items" if you want (possibly also clearing items from other).